ChangeLog.md

```diff
@@ -95,6 +95,7 @@
 * Ability to Get-AzSqlServerServiceObjective by location without needing a preexisting server in the region.
 * Support for time zone parameter in Managed Instance create.
 * Fix documentation for wildcards
+* Support of Serverless specific parameters
```

Comment:
Please do not add changes to the root changelog

Reply:
Deleted this part from root changelog